### PR TITLE
[bitnami/elasticsearch] add priority class name for all es sts

### LINKED ISSUE #7702: [bitnami/elasticsearch] Add support for priorityClassName
**Which chart**:
elasticsearch

**Is your feature request related to a problem? Please describe.**
It is not possible to define priorityClassName for master and data statefulsets

### PR DESCRIPTION
**Pull request by cyril-corbon**
**Description of the change**
add priority class name on all sts 

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #7702

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Analytics
 apiVersion: v2
-appVersion: 7.15.1
+appVersion: 7.16.1
```
Comment (alemorcuq) [inline bitnami/elasticsearch/Chart.yaml:4]: This is the Elasticsearch version in the container. No need to change it.

File: bitnami/elasticsearch/Chart.yaml [hunk 2]
```diff
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/bitnami-docker-elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 17.1.8
+version: 17.2.8
```
Comment (alemorcuq) [inline bitnami/elasticsearch/Chart.yaml:28]: `17.2.0` would be the next minor.

```suggestion
version: 17.2.0
```

File: bitnami/elasticsearch/templates/coordinating-statefulset.yaml [hunk 3]
```diff
@@ -44,6 +44,7 @@ spec:
       {{- if .Values.coordinating.schedulerName }}
       schedulerName: {{ .Values.coordinating.schedulerName }}
       {{- end }}
+      priorityClassName: {{ .Values.coordinating.priorityClassName | quote }}
```
Comment (alemorcuq) [inline bitnami/elasticsearch/templates/coordinating-statefulset.yaml:47]: This should be wrapped in an if statement to check if `priorityClassName` is set. Like this: 

```suggestion
      {{- if .Values.coordinating.priorityClassName }}
      priorityClassName: {{ .Values.coordinating.priorityClassName | quote }}
      {{- end }}
```

The same applies to the other statefulsets.

File: bitnami/elasticsearch/values.yaml [hunk 4]
```diff
@@ -298,6 +298,10 @@ master:
   ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
   ##
   affinity: {}
+  ## @param master.priorityClassName Priority Class Name
```
Comment (alemorcuq) [inline bitnami/elasticsearch/values.yaml:301]: I would be more specific in the description, like:

```suggestion
  ## @param master.priorityClassName Master pods Priority Class Name
```

The same applies to the rest. Also, please double check the text is the same as in the README file.

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left some comments, please check them.

**Comment by cyril-corbon** [comment]
review applied :smile:

**Review by alemorcuq — APPROVED** [review]
Great, thanks!

**Pull request merged**